src/ops/ops.go

```diff
@@ -922,6 +922,11 @@ func (o *ops) OverwriteOsImage(osImage, device string, extraArgs []string) error
 	} else {
 		growpartcmd = makecmd("growpart", device, "4")
 	}
+	ret, err := o.ExecPrivilegeCommand(nil, "uname", "-m")
```

Comment:
I wonder why we don't run this validation here https://github.com/openshift/assisted-installer/blob/af293d6bed78f1488fa2e323b029aaca4c081de5/src/installer/installer.go#L376 ?

Reply:
Thank you for the comment. Due to the fact that this is a backport and the issue in/with ostree/zipl need to be investigated and solved asap I decided to put the check as close as possible to the "problem".